packages/dev/docs/pages/blog/building-a-numberfield.mdx

```diff
@@ -0,0 +1,233 @@
+</div>
+<br />
+
+Number fields shouldn't allow non-valid characters in them, so we run validation before every change to the field.
+If the incoming change results in something we can't parse, then we block that change.
+Number fields supporting multiple numbering systems also need to support input method implementations (IMEs) such as Pinyin.
+We do this through composition events.
+During a composition event, we stop running our validation and start again when the composition event ends.
+
+<Video src={pinyinIMEentering} loop autoPlay muted style={{margin: '0px auto'}} />
+
+## Problems with input type number
+
+A native number field, or input type=number, cannot accept any characters except for latin numerals, minus and plus signs, decimals, and the letter e for exponentials. It will prevent the input of other characters such as would be used for units or currency.
+It will also block characters that belong to other numbering systems.
+As such, they cannot handle the kind of formatting and internationalization we need to support.
+Native number fields have historically had not great accessibility, which is nicely documented in this post.
+[Why the GOV.UK Design System team changed the input type for numbers](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/)
+Many of these have been fixed now, but it's worth pointing out that some where fixed during the course of writing this component, and some have not yet been addressed.
+The way in which VoiceOver announces changes in values to a number field is still problematic, for example, not reading off the new value when the input in changed, only after leaving and coming back to the field, as shown here. What VoiceOver reads is easiest to listen to, but it is also displayed at the bottom of the screen.
+<div style={{display: 'flex', width: '100%', overflow: 'hidden'}}>
+  <Video src={nativeNumberInputVO} style={{flex: '1 1 0', minWidth: '0', margin: '0 100px'}} />
+  <Video src={numberFieldVO} style={{flex: '1 1 0', minWidth: '0', margin: '0 100px'}} />
+</div>
+
+Native number fields vary a lot between browsers.
+As mentioned above, there are differences in the mobile keyboard that is used.
+They also have UI differences, for instance, the steppers.
+Some browsers allow you to control when they appear and when they are hidden and none of the browsers spin buttons are a great experience on mobile.
+Browsers also vary on the allowed characters in a native input type=number and if the scroll wheel can also step values.
+A good place to see these differences in UI is this [codepen](https://codepen.io/2ne/pen/GzovC).
+
+
+## Internationalization
+
+### Numeral systems vs locales
+In our NumberField, we support three [numeral systems](https://en.wikipedia.org/wiki/Numeral_system) out of the browsers [Intl numbering system](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/numberingSystem).
```

Comment:
We only support the three because we only check those three https://github.com/adobe/react-spectrum/blob/main/packages/%40internationalized/number/src/NumberParser.ts#L24
we could probably open it to the others, but we haven't tried to yet